DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: cleanepi
 Title: Clean and Standardize Epidemiological Data
-Version: 1.1.0
+Version: 1.1.0.9000
```

Comment:
I thought the development version is created after I run `usethis::use_dev_version(push = TRUE)` (which I have not done yet)?

Reply:
Yes, I just did it in this PR since I was making changes along these lines.

Reply:
I see - Thanks the review.
I will create the github release and run that command once I finished drafting the blogpost.

Reply:
I have created the GitHub release since the CRAN-RELEASE file was created on my local computer when I uploaded to CRAN

Reply:
sounds good with me.